docs/sources/whatsnew/whats-new-in-v8-3.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "What's new in Grafana v8.3"
+description = "Feature and improvement highlights for Grafana v8.3"
+keywords = ["grafana", "new", "documentation", "8.3", "release notes"]
+weight = -33
+aliases = ["/docs/grafana/latest/guides/whats-new-in-v8-3/"]
+[_build]
+list = false
++++
+
+# What’s new in Grafana v8.3
+
+Grafana 8.3 is an exciting release for Grafana Labs. This release includes the new Candlestick Panel, a new visualization suggestions engine and, for enterprise users, Recorded Queries. 
+
+For Open Source users it also marks the first time Grafana Alerting, formerly unified alerting, is enabled by default for new Grafana installations. Grafana Alerting in 8.3 is the flexible, single pane of glass for all your alerts. Included in this release is expanded provisioning support for notifiers, contact points, and alert rules, alongside auditing and fine-grained access control for our Enterprise customers.
+
+We’ve summarized what’s new in the release here, but you might also be interested in the announcement blog post as well. If you’d like all the details you can check out the complete [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+
+# Grafana OSS
+
+## Accessibility
+
+We’ve continued to make progress on improving Grafana’s accessibility. In Grafana 8.3 we’ve updated the main menu with improved keyboard navigation. We’ve also improved Grafana’s behavior when navigating through elements with the tab key (tab stops) and finished our work to make Grafana’s viewer roles compatible with assistive technologies such as screen readers. You can read our accessibility statement here and reach out to us with accessibility issues using our community Slack or our community forums. 
+
+## Dashboards and Visualizations
+
+### Panel Suggestions
+
+Grafana will now make suggestions for visualization types based on the current data surfaced by a query or queries in a panel. This makes seeing available, possible interpretations of your data more straightforward and can be a useful jumping-off point when building panels and dashboards with specific goals in mind. 
+
+### Candlestick Panel (Beta)
+
+Grafana 8.3 is shipping with a new Candlestick panel that is so much more. Beyond candlesticks and open, high, low, high, close, behaviors, it includes customizable up/down colors, bar color determined by intra-period or inter-period movement of the data, volume histogram with matching colors, and the ability to detach or create a separate volume histogram to allow for more flexible dashboard design. You can read more about it here [link to do].
+
```

Comment:
geomap improvements:

* faster loading (when color+size are fixed)
* custom markers (svg icons)
* markers + text
* multiple layers in one map
* static geojson overlays